src/app/reskin/components/tabs/nav-tabs.tsx

```diff
@@ -10,16 +10,22 @@ export const NavTabs = ({
   return (
     <div className="fixed bottom-0 sm:sticky dark:bg-reskin-dark-gray w-full bg-white">
       <div className="flex h-20 flex-row justify-between items-center w-full sm:mx-auto max-w-5xl relative z-10">
-        {tabs.map((tab, index) => (
-          <Link
-            className={`h-full uppercase flex-1 flex justify-center items-center ${pathname === tab.link ? "border-t-orange sm:border-b-orange dark:border-t-dark-reskin-orange font-bold dark:sm:border-b-dark-reskin-orange" : "border-t-transparent sm:border-b-[#D9D9D9]"} border-t-4 sm:border-b-4 sm:border-t-0 sm:bottom-auto`}
-            key={index}
-            href={tab.link}
-            prefetch={true}
-          >
-            {tab.label}
-          </Link>
-        ))}
+        {tabs.map((tab, index) => {
+          const [, tabNameSpace] = tab.link.split("/");
+          const [, pathNameSpace] = pathname.split("/");
```

Comment:
This will work only if the first element is unique in the tab collection, but for the (current) reskin that is not the case (eg, `/reskin/withdraw` vs `/reskin`). Is it fine?

Reply:
fair nuff